plugins/nodes/src/js/components/LeaderGrid.js

```diff
@@ -18,6 +19,14 @@ function timeFromNow(time) {
   return moment(time * 1000).fromNow();
 }
 
+function formattedDate(time) {
```

Comment:
What about refactoring both time functions to a time util. I think in the future, it will help a lot to deal consistently with time across DC/OS UI.

Reply:
Agree.

Reply:
there actually already is `DateUtil.msToDateStr`, but its output looks weird: `
08-28-2018 at 9:17am` 🤔

Reply:
Looks like the only place that existing date util function is being used is in the Nodes Details tab. Maybe it would make sense to have the dates in the same format in both places? Also it looks like there is a `DateUtil.msToRelativeTime` that maybe could replace `timeFromNow` here?